src/utils/SoundSingleton.ts

```diff
@@ -24,6 +26,7 @@ export class SoundSingleton {
   public soundEffectsEnabled = true
   public musicVolume = 3
   public soundEffectsVolume = 3
+  public area = ""
```

Comment:
I think the name of this property could be more descriptive, something like `playerCurrentArea`

Reply:
Cool! Done